### DISCUSSION TITLE
1.0.4

### DISCUSSION THREAD

**Discussion by mrlika** [Novage/p2p-media-loader#402] [Releases]
## What's Changed
* Fix Clappr demo
* Refactoring peer protocol. The P2P protocol version changed to `v2`
* Fix  Safari tracker announcement (multiple WebSockets) issue
* Update documentation
* Extend interfaces with the new events

<hr /><em>This discussion was created from the release <a href='https://github.com/Novage/p2p-media-loader/releases/tag/1.0.4'>1.0.4</a>.</em>

**Comment by Goppi100** [comment]
> ## What's Changed
> * Fix Clappr demo
> * Refactoring peer protocol. The P2P protocol version changed to `v2`
> * Fix  Safari tracker announcement (multiple WebSockets) issue
> * Update documentation
> * Extend interfaces with the new events
> 
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/Novage/p2p-media-loader/releases/tag/1.0.4'>1.0.4</a>.</em>
